src/test/regress/expected/create_distributed_table_concurrently.out

```diff
@@ -0,0 +1,238 @@
+create schema create_distributed_table_concurrently;
+set search_path to create_distributed_table_concurrently;
+set citus.shard_replication_factor to 1;
+-- make sure we have the coordinator in the metadata
+SELECT 1 FROM citus_set_coordinator_host('localhost', :master_port);
+ ?column?
+---------------------------------------------------------------------
+        1
+(1 row)
+
+create table ref (id int primary key);
+select create_reference_table('ref');
+ create_reference_table
+---------------------------------------------------------------------
+
+(1 row)
+
+insert into ref select s from generate_series(0,9) s;
+create table test (key text, id int references ref (id) on delete cascade, t timestamptz default now()) partition by range (t);
+create table test_1 partition of test for values from ('2022-01-01') to ('2022-12-31');
+create table test_2 partition of test for values from ('2023-01-01') to ('2023-12-31');
```

Comment:
maybe also add another test with a columnar partition (may have to be a separate table due to foreign key)